github.go

```diff
@@ -157,3 +197,57 @@ func (g *GitHubPullRequest) comment() ([]*github.PullRequestComment, error) {
 	}
 	return comments, nil
 }
+
+// ---
+// GitHub PullRequest Review API Implementation
+// ref: https://github.com/google/go-github/issues/495
+
+const (
+	mediaTypePullRequestReview = "application/vnd.github.black-cat-preview+json"
+)
+
+type Review struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Review should have comment or be unexported